src/styles/mixins/buttons.scss

```diff
@@ -18,6 +18,7 @@
  * And pass 'my-component-button' as the `className` prop to `Button`.
  */
 @use "@hypothesis/frontend-shared/styles/mixins/focus";
+@use "@hypothesis/frontend-shared/styles/mixins/utils" as u;
```

Comment:
collides with @use "./utils"; so I named space with `u`, probably a bad choice, but it was terse.

Reply:
How about `frontend-utils` or similar? Single-letter references are a pain :)

Reply:
Yes, for now, since I don't be adding the transition, we don't ned to cross that bridge, but when we do thats a reasonable name.